custodia.spec

```diff
@@ -1,25 +1,12 @@
-%if 0%{?fedora}
-%global with_python3 1
-%endif
-
 %{!?version: %define version 0.6.dev1}
 
-# Workaround for python-etcd issue on PPC64. Although it's a noarch package
-# it depends on etcd for testing. Go does not support PPC64 yet.
-%ifarch ppc64
-%global with_etcd 0
-%else
-%global with_etcd 1
+%if 0%{?fedora}
```

Comment:
Maybe %if 0%{?fedora} >= 26 here so if someone tries to build on f25 he'll get only py2 bits ?